config/default.json

```diff
@@ -24,5 +24,8 @@
         "options": null
       }
     }
-  }
+  },
+  "layerSourceDir": "/layerSources",
+  "displayNameDir": "\\layerSources",
```

Comment:
please add section for storage explorer:
```
storageExplorer": {
    "displayNameDir": ...
}
```

Reply:
done